assets/manifest.json

```diff
@@ -22,7 +22,8 @@
     {
       "matches": [
         "https://github.com/*",
-        "https://gist.github.com/*"
+        "https://gist.github.com/*",
+        "https://*/*"
```

Comment:
The new pattern would match any URL that uses https, which shouldn't be allowed since not every https url is a github instance.